src/docs/release-notes/R23/FullNotes.mdx

```diff
@@ -31,3 +36,18 @@
 -   Bug fixes & minor updates
 
 ### Package Summary
+
+#### Icons
+
+-   @brightlayer-ui/icons v1.9.0 ([Changelog](https://github.com/brightlayer-ui/icons/blob/master/CHANGELOG.md))
+-   @brightlayer-ui/icons-svg v1.9.0 ([Changelog](https://github.com/brightlayer-ui/icons/blob/master/CHANGELOG.md))
+-   @brightlayer-ui/icons-mui v3.0.0 ([Changelog](https://github.com/brightlayer-ui/icons/blob/master/mui/CHANGELOG.md))
+-   @brightlayer-ui/react-native-vector-icons v1.5.0 ([Changelog](https://github.com/brightlayer-ui/icons/blob/master/rn-vector/CHANGELOG.md))
+
+#### Angular
+
+-   @brightlayer-ui/angular-components v7.0.0 ([Changelog](https://github.com/brightlayer-ui/angular-component-library/blob/master/CHANGELOG.md))
+
+#### React
+
+-   @brightlayer-ui/react-components v6.0.0 ([Changelog](https://github.com/brightlayer-ui/react-component-library/blob/master/CHANGELOG.md))
```

Comment:
Let's add items for React Themes, React Workflows, React Templates

Reply:
For react templates, do we list all six packages